delegation-toolkit/concepts/advanced-permissions.md

```diff
@@ -23,42 +23,43 @@ Dapps can use this method to request a wallet to grant the dapp permission to ex
 `wallet_grantPermissions` requires a `signer` parameter, which identifies the entity requesting or managing the permission.
 Common signer implementations include wallet signers, single key and multisig signers, and account signers.
 
-The Smart Accounts Kit supports multiple types of signers, but [an account signer is used in this documentation's examples](../guides/erc7715/execute-on-metamask-users-behalf.md) as a common implementation.
-With an account signer, a session account is created and used for the single purpose of requesting and redeeming ERC-7715 permissions, and does not contain tokens.
+The Smart Accounts Kit supports multiple types of signers, but [an account signer is used in this documentation's examples](../guides/advanced-permissions/execute-on-metamask-users-behalf.md) as a common implementation.
```

Comment:
```suggestion
Smart Accounts Kit supports multiple signer types. The documentation uses [an account signer](../guides/advanced-permissions/execute-on-metamask-users-behalf.md) as a common implementation example.
```

Reply:
@bgravenorst @AyushBherwani1998 Btw, we should come to a consensus on how we refer to this kit. In the [rebrand PR](https://github.com/MetaMask/metamask-docs/pull/2421), we refer to it as "the Smart Accounts Kit" where applicable, instead of just "Smart Accounts Kit." I think this makes more sense, as it replaces "the Delegation Toolkit."